pkg/engine/validation_test.go

```diff
@@ -31,195 +31,195 @@ func TestGetAnchorsFromMap_ThereAreAnchors(t *testing.T) {
+//							"spec":{
+//								"ports":[
+//									{
+//										"name":"hs",
+//										"protocol":32
+//									}
+//								]
+//							}
+//						}
+//					}
+//				}
+//			]
+//		}
+//	}`)
+//	rawResource := []byte(`{
+//		"kind":"Service",
+//		"apiVersion":"v1",
+//		"metadata":{
+//			"name":"game-service",
+//			"labels":{
+//				"originalLabel":"isHere",
+//				"secretLabel":"thisIsMySecret"
+//			}
+//		},
+//		"spec":{
+//			"selector":{
+//				"app":"MyApp"
+//			},
+//			"ports":[
+//				{
+//					"name":"http",
+//					"protocol":"TCP",
+//					"port":80,
+//					"targetPort":9376
+//				}
+//			]
+//		}
+//	}
+//	`)
+//
+//	var policy kyverno.ClusterPolicy
+//	json.Unmarshal(rawPolicy, &policy)
+//
+//	resourceUnstructured, err := utils.ConvertToUnstructured(rawResource)
+//	assert.NilError(t, err)
+//
+//	er := Validate(PolicyContext{Policy: policy, NewResource: *resourceUnstructured})
+//	assert.Assert(t, len(er.PolicyResponse.Rules) == 0)
+//}
+//
+//func TestValidate_MapHasFloats(t *testing.T) {
+//	rawPolicy := []byte(`{
+//		"apiVersion":"kyverno.nirmata.io/v1",
+//		"kind":"ClusterPolicy",
+//		"metadata":{
+//			"name":"policy-deployment-changed"
+//		},
+//		"spec":{
+//			"rules":[
+//				{
+//					"name":"First policy v2",
+//					"resource":{
+//						"kinds":[
+//							"Deployment"
+//						],
+//						"name":"nginx-*"
+//					},
+//					"mutate":{
+//						"patches":[
+//							{
+//								"path":"/metadata/labels/isMutated",
+//								"op":"add",
+//								"value":"true"
+//							},
+//							{
+//								"path":"/metadata/labels/app",
+//								"op":"replace",
+//								"value":"nginx_is_mutated"
+//							}
+//						]
+//					},
+//					"validate":{
+//						"message":"replicas number is wrong",
+//						"pattern":{
+//							"metadata":{
+//								"labels":{
+//									"app":"*"
+//								}
+//							},
+//							"spec":{
+//								"replicas":3
+//							}
+//						}
+//					}
+//				}
+//			]
+//		}
+//	}`)
+//	rawResource := []byte(`{
+//		"apiVersion":"apps/v1",
+//		"kind":"Deployment",
+//		"metadata":{
+//			"name":"nginx-deployment",
+//			"labels":{
+//				"app":"nginx"
+//			}
+//		},
+//		"spec":{
+//			"replicas":3,
+//			"selector":{
+//				"matchLabels":{
+//					"app":"nginx"
+//				}
+//			},
+//			"template":{
+//				"metadata":{
+//					"labels":{
+//						"app":"nginx"
+//					}
+//				},
+//				"spec":{
+//					"containers":[
+//						{
+//							"name":"nginx",
+//							"image":"nginx:1.7.9",
+//							"ports":[
+//								{
+//									"containerPort":80
+//								}
+//							]
+//						}
+//					]
+//				}
+//			}
+//		}
+//	}
+//	`)
+//
+//	var policy kyverno.ClusterPolicy
+//	json.Unmarshal(rawPolicy, &policy)
+//
+//	resourceUnstructured, err := utils.ConvertToUnstructured(rawResource)
+//	assert.NilError(t, err)
+//	er := Validate(PolicyContext{Policy: policy, NewResource: *resourceUnstructured})
+//	assert.Assert(t, len(er.PolicyResponse.Rules) == 0)
+//}
```

Comment:
If the test is no longer valid, please remove the code.

Reply:
Alright